blogposts/2021/release-3.29.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Sourcegraph 3.29 release"
+publishDate: 2021-06-18T10:00-07:00
+description: "The Sourcegraph 3.29 release introduces improved search results ranking, makes it easier to create Code Insights, and includes support for bulk actions with Batch Changes."
+tags: [blog, release]
+slug: "release/3.29"
+published: false
+changelogItems:
+  - description: "Code Insights queries can now run concurrently up to a limit set by the `insights.query.worker.concurrency` site config."
+    url: https://github.com/sourcegraph/sourcegraph/pull/21219
+    category: Code Insights
+  - description: "Code Insights workers now support a rate limit for query execution and historical data frame analysis using the `insights.query.worker.rateLimit` and `insights.historical.worker.rateLimit` site configurations."
+    url: https://github.com/sourcegraph/sourcegraph/pull/21533
+    category: 
+  - description: "The GraphQL `Site` `SettingsSubject` type now has an `allowSiteSettingsEdits` field to allow clients to determine whether the instance uses the `GLOBAL_SETTINGS_FILE` environment variable."
+    url: https://github.com/sourcegraph/sourcegraph/pull/21827
+    category: API
+  - description: "Code Insights creation UI now has auto-save logic and clear all fields functionality"
```

Comment:
I agree with Joel, and that anything mentioned in the highlights should have the short summary in the changelog below.

Reply:
@christinaforney @Joelkw - we can definitely keep this in the release post. 

Previously, we had the release post (blog) and the changelog. The guidance from Christina was that the release post included a curated list of updates and the changelog was comprehensive and included everything. 

@Joelkw - are you suggesting that the release post should still include every single update? If yes, then should we just link to the changelog?

Reply:
@abschu I got similar guidance from Christina that I used for the 3.23 and 3.24 release posts and I think "curated list of updates" means what this post currently is without the suggested removals you commented: it contains a curated list of updates in the changelog section of the blog post (rather than the entire changelog from changelog.md). In addition to that "curated list" (again, in the changelog section of a release bog post), we pull out items we want to add images/gifs/videos of and add description. 

I don't think anyone would care strongly if you wanted to also link to the changelog.md itself (possibly an argument we shouldn't call anything in the blog post "changelog" if it's not complete), but we'd need a way to communicate/name the highlights we pulled out into the blog post list and why they're different.

Reply:
That sounds good - happy to leave these items in here.